src/pages/validator/amplifier/verifier-operations.mdx

```diff
@@ -0,0 +1,46 @@
+# Verifier operations
+
+import { Callout } from "/src/components/callout"
+
+<Callout>
+The Axelar Virtual Machine (AVM) and Amplifier are currently under active development, so these instructions are likely to change. Please check back frequently for updates.
+</Callout>
+
+## Approvals
+
+### Approve a verifier or verifier set
+
+Each verifier in a verifier set must use `ampd` to bond and register chain support themselves. On testnet and mainnet, community governance will call the following to authorize them:
+
+```bash
+axelard tx wasm execute [service registry address] '{"authorize_workers":{"workers":[list of worker addresses], "service_name":[service-name]}}' --from key --gas auto --gas-adjustment 2
+```
+
+## Rewards
+
+There is one [rewards contract](https://github.com/axelarnetwork/axelar-amplifier/blob/main/doc/src/contracts/rewards.md) for all chain integrations. Rewards are paid out in $AXL.
+
+### How rewards are calculated
+
+Every action earns rewards within `n` epochs. `n` more epochs will be paid out based on what has already been paid. There is a single global parameter for each epoch called `rewards_per_epoch`, in `Uint128`. Later, there will be one per pool. 
+
+A [threshold of weighted votes](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/voting-verifier/src/msg.rs#L25) is required for voting to be considered complete for a particular message. Verifiers must hit that threshold for payout.
```

Comment:
I'm confused. The voting threshold is independent from payouts. Maybe we mean a vote must pass threshold for the verifiers to qualify for `RecordParticpiation`? I'm not sure that's true?

Reply:
```suggestion
A [threshold of weighted votes](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/voting-verifier/src/msg.rs#L25) is required for voting to be considered complete for a particular message.
```